packager/mpd/base/mpd_utils.cc

```diff
@@ -324,6 +325,10 @@ const char kMarlinUUID[] = "5e629af5-38da-4063-8977-97ffbd9902d4";
 // Unofficial FairPlay system id extracted from
 // https://forums.developer.apple.com/thread/6185.
 const char kFairPlayUUID[] = "29701fe4-3cc7-4a34-8c5b-ae90c7439a47";
+// String representation of media::kPlayReadySystemId
+const char kPlayReadyUUID[] = "9a04f079-9840-4286-ab92-e65be0885f95";
+
```

Comment:
Add a link on the reason?

For example:

// It is RECOMMENDED to include the @value attribute with name and version "MSPR 2.0".
// See https://docs.microsoft.com/en-us/playready/specifications/mpeg-dash-playready#221-general.

Reply:
Of course, forgot it, thanks )